### DISCUSSION TITLE
Waypoints overprinted

### DISCUSSION THREAD

**Discussion by normie555** [flybywiresim/aircraft#4537] [Help] [answered]
![double3](https://user-images.githubusercontent.com/71556913/115750768-63605900-a34d-11eb-9f9f-07b77f7203c1.png)

Halfway through a flight, I lose the track line, and waypoints now are shown twise, once lower sase and upper case on top
Is anyone else experiencing this

**Comment by Paytheon** [comment]
You should also screenshot the VFR Map to. What I have found is the VFR map shows your plane is still on the flight plan, but the HD seems to have lost track of the lines.

**Reply by normie555** [reply]
There is no problem with the autopilot following the flightplan, I continued the entire flight OK. This is only a display issue and I want to know if it is only produced on my system

**Comment by hotshotp** [comment, marked as answer]
known issue

**Comment by normie555** [comment]
Thanks confirming not just me
